qt/DetailsDialog.cc

```diff
@@ -386,6 +390,40 @@ void DetailsDialog::onSessionCalled(Session::Tag tag)
     }
 }
 
+void DetailsDialog::onButtonBoxClicked(QAbstractButton* button)
+{
+    if (ui_.dialogButtons->standardButton(button) == QDialogButtonBox::Close)
+    {
+        if (ui_.labelsTextEdit->isReadOnly()) // mixed -> no edits could have been made
```

Comment:
_(nit)_ could also happen if there are no torrents.

```suggestion
        if (ui_.labelsTextEdit->isReadOnly()) // no edits could have been made
```

Reply:
done